prestoadmin/util/httpscacertconnection.py

```diff
@@ -0,0 +1,36 @@
+import socket
+import ssl
+import httplib
+
+# Adapted from http://code.activestate.com/recipes/577548-https-httplib-client-connection-with-certificate-v/
+# BSD-licensed.
+
+
+class HTTPSCaCertConnection(httplib.HTTPSConnection):
+    """ Class to make a HTTPS connection, with support for full client-based SSL Authentication"""
```

Comment:
No, we need this because 2.6.x and < 2.7.9 don't support it. This sets up the ca-certs (using the one from the private key) so that cert validation passes on new versions of python, while *also* working on old versions.